### PR TITLE
Finish ticket 5 plus add loading

### PR DESCRIPTION
**Pull request by BrokenRecord85**

### CODE REVIEW

**Review by leekli — CHANGES_REQUESTED** [review]
Removing my original comment as it wasn't applicable.

**Comment by BrokenRecord85** [comment]
But that's ticket 6-view an individual review, I haven't done that yet,

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

File: src/utils/api.js [hunk 1]
```diff
@@ -5,8 +5,9 @@ const gamesApi = axios.create({
 }) 
 
 
-export const getReviews = () => {
-    return gamesApi.get('/reviews').then((res) => {
+export const getReviews = (category) => {
+    console.log(category, 'API heeeeeeeeeeeeeeeeeeeeeeeerrree')
```
Comment (leekli) [inline src/utils/api.js]: Let's remove this console log

**Review by leekli — CHANGES_REQUESTED** [review]
Apologies Pawel I was reading the wrong thing!

OK so I can see most of ticket 5 is implemented, one thing I'd like to see if some 'be some UI elements to navigate to each categories pages (links / dropdown etc.)' rather than having to only click the category under each article.

So some sort of component/page/list/drop down/whatever so a user can easily see a list of categories and click any to get them. Currently they would have to scroll through your reviews to try and find the category name they're looking for. 

Please have a look at implementing this, then submit a PR tomorrow for review

File: src/components/Categories.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+import { useState, useEffect,  } from 'react'
+import { getCategories } from '../utils/api'
+import { useNavigate, useParams } from 'react-router-dom'
+
+const Categories = () => {
+  const [categories, setCategories] = useState([])
+  const [selectedCategory, setSelectedCategory] = useState('all')
+  const navigate = useNavigate()
+  const params = useParams()
+  console.log(params.category)
```
Comment (leekli) [inline src/components/Categories.jsx:11]: This console log to remove

File: src/components/Categories.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    }
+    
+  }
+  
+  return (
+    <select className='select-cat' onChange={handleChange} value={selectedCategory}  name="categoriesList" id="categories-list">
+        <option value='' defaultValue>Select category: </option>
+        <option value="all_categories">All Categories</option>
+        {categories.map((category, index, array) => {
+          //console.log(array)
```
Comment (leekli) [inline src/components/Categories.jsx:35]: Remove this line

**Review by leekli — APPROVED** [review]
Hi, as we've just been looking at this for an hour I'm happy to approve.

I've left 2 comments of lines which need removing through. You're free to merge.

**Pull request merged**